Refine tctx and rctx claim handling in Txn-Token (#361)
oauth-wg/oauth-transaction-tokens: draft-ietf-oauth-transaction-tokens.md

* The TTS MUST set the `txn` claim to a unique ID specific to this transaction.
* The TTS MAY set the `iss` claim of the Txn-Token to a value defining the entity that signed the Txn-Token.
* The TTS MUST evaluate the value specified in the `scope` parameter of the request to determine the `scope` claim of the issued Txn-Token.

> * If a `request_context` parameter is present in the Txn-Token Request, the TTS SHOULD evaluate the content of the `request_context` parameter and may include claims in the `rctx` claim of the issued Txn-Token based on the `request_context` parameter. The claims included in the `rctx` claim of the issued Txn-Token may be an exact match of those on the `request_context`, or may be derived from the `request_context`. The TTS is authoritative for the set of claims made available in the `rctx` claim and MAY include additional claims unrelated to the `request_context`.

Reviewer: "exact match or derived from request_context" may not be sufficient. The TTS might decide to add values of its own in the rctx". Perhaps we can say "exact match, derived from the request_context or independently asserted by the TTS"

Author: I think that is what this sentence tries to do:

"The TTS is authoritative for the set of claims made available in the `rctx` claim and MAY include additional claims unrelated to the `request_context`."

But I like your suggestion as it is more concose and all in the same sentence.

> * If a `request_context` parameter is present in the Txn-Token Request, the TTS SHOULD evaluate the content of the `request_context` parameter and may include claims in the `rctx` claim of the issued Txn-Token based on the `request_context` parameter. The claims included in the `rctx` claim of the issued Txn-Token may be an exact match of those on the `request_context`, or may be derived from the `request_context`. The TTS is authoritative for the set of claims made available in the `rctx` claim and MAY include additional claims unrelated to the `request_context`.

Author: ```suggestion
* If a `request_context` parameter is present in the Txn-Token Request, the TTS SHOULD evaluate the content of the `request_context` parameter and may include claims in the `rctx` claim of the issued Txn-Token based on the `request_context` parameter. The TTS is authoritative for the set of claims made available in the `rctx` claim. The claims included in the `rctx` claim of the issued Txn-Token may be an exact match of those on the `request_context`, may be derived from the `request_context`, or independently asserted by the TTS.
```
